gtsfm/loader/hilti_loader.py

```diff
@@ -120,12 +235,33 @@ def __read_lidar_pose_priors(self) -> Dict[int, Pose3]:
 
         return w_T_imu
 
+    def __read_fastlio_poses(self) -> List[Pose3]:
+        """Read the poses for the IMU for rig indices."""
+        filepath = str(self._base_folder / GT_POSE_RELATIVE_PATH)
+
+        def pose_of_vector(pose_vector):
+            """Convert from vector to Pose3"""
+            pose_rotation = Rot3(pose_vector[-1], pose_vector[3], pose_vector[4], pose_vector[5])
```

Comment:
is this a quaternion? maybe add comment.

Reply:
Added